web/app/gospeak/web/GsApplicationLoader.scala

```diff
@@ -113,7 +123,16 @@ class GsComponents(context: ApplicationLoader.Context)
       sharedSecret = configuration.underlying.getString("silhouette.jwt.authenticator.sharedSecret"))
     new JWTAuthenticatorService(config, None, authenticatorDecoder, idGenerator, clock)
   } */
-
+  lazy val youtubeClient: YoutubeClient = {
```

Comment:
It could be nice to put all this setup inside the YoutubeClient class (in a factory method), so it won't pollute this (big) class ^^